pkg/install/generate.go

```diff
@@ -211,11 +170,14 @@ func GenerateInstallerJob(
 	containers := []corev1.Container{
 		{
 			Name:            "installer",
-			Image:           installerImage,
-			ImagePullPolicy: installerImagePullPolicy,
+			Image:           releaseImage,
+			ImagePullPolicy: releaseImagePullPolicy,
 			Env:             env,
 			Command:         []string{"/bin/sh", "-c"},
 			Args:            []string{"cp -v /bin/openshift-install /output && ls -la /output"},
+// FIXME: should this code or the remote container find and execute the installer container?
+// $ oc adm release info --image-for installer registry.svc.ci.openshift.org/openshift/origin-release:v4.0
+// registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-02-05-150550@sha256:33a38e5e93d29e283ca93da8dde370b792b128181deac4b7e0d3c78e473deccd
```

Comment:
This is going to be a bit tricky, the extracting from a release image is a bit time consuming. That feels like it should not be in generate.go, which will soon be getting run more frequently on sync to make sure the install job looks like it should (if the cd changes it will be updated and restarted). We wouldn't want to repeatedly do this.

Can a container execute a container?

CC @csrwng

Reply:
> We wouldn't want to repeatedly do this

On this front, if wherever you're pulling it has a layer cache, this should just be "check the tag, and, good, I already have that manifest and an unpacked image filesystem" (like pull-always) unless the tag had shifted, and you'd want the re-pull then anyway.  If we're willing to commit to not clobbering tags (e.g. not using the floating registry.svc.ci.openshift.org/openshift/origin-release:v4.0), then yeah, we could cache the resolved installer payload in Go.

Reply:
> Can a container execute a container?

@dgoodwin not sure how it would be needed in this case

Reply:
Referring to the comment above re a remote container finding and executing the installer container. 

If we do it in controllers it could be slow, if we do it in a pod we're just doing a lookup for the install container to run another pod with.

Reply:
Maybe a controller maintaining a configmap of release image to extracted installer image, for every release image that shows up in a clusterdeployment. Install job can't start until our item shows up there.

Reply:
Would a release image be required now?

Reply:
> Would a release image be required now?

No, I set the same OKD default master installer builds use.  But those are garbage-collected after 72 hours, so you probably want to set an update payload (or change the default).